scene/animation/animation_player.cpp

```diff
@@ -1241,6 +1241,26 @@ void AnimationPlayer::stop(bool p_reset) {
 	playing = false;
```

Comment:
Wasn't `stop()` the cause of the issue in the first place? But you haven't changed it here so I am assuming the bug reported in the linked issue still exists?